### DISCUSSION TITLE
请求添加模拟器 ppsspp

### DISCUSSION THREAD

**Discussion by mokurin000** [Gamuxorg/bbs#264] [投递和请求游戏]
游戏名称：ppsspp
游戏名称(英文)：ppsspp
下载地址 :（如果有的话）仓库/[官网](https://www.ppsspp.org/)

**Comment by shouhuanxiaoji** [comment]
好的，尽快收录，感谢

**Comment by ghgl2003** [comment]
直接玩3dS模拟器吧，有linux3ds的模拟器，3ds上的游戏 也比ppsspp好玩

**Comment by hyh19962008** [comment]
已添加
